src/Application.php

```diff
@@ -27,6 +27,11 @@ public function __construct(LoggerInterface $logger)
         $config['parameters'] = $config['parameters'] ?? [];
         $config['parameters']['data_dir'] = $dataDir;
 
+        if (isset($config['image_parameters']['global_config']['absCredentialsType'])) {
+            $config['parameters']['absCredentialsType'] =
```

Comment:
Pridaj to prosim do README.

Navrhujem skor default value ako prepisovat ... kedze predpokladam, ze sa to bude este rozne testovat, a ked to prepises, tak to uz nebude mozne spustit so SAS tokenom na otestovanie.

Reply:
jj myslel jsem že nechtějí spouštět se SAS tokenem ale jen přes managed identity, ale ok můžem to dát jako default value https://github.com/keboola/db-writer-synapse/pull/6/commits/e6d6d1d3f7a516813d46afbae6252c36c277a274

Reply:
ještě jsem přidal replace test https://github.com/keboola/db-writer-synapse/pull/6/commits/ebdf537ca7c667cdfd15a93f959d407a3aa5427c

Reply:
a readme https://github.com/keboola/db-writer-synapse/pull/6/commits/bec9c2f754a503424487ec675a489280c1deb70b